src/main/java/org/sonatype/gradle/plugins/scan/common/DependenciesFinder.java

```diff
@@ -71,13 +74,12 @@
 
   private static final String ATTRIBUTES_SUPPORTED_GRADLE_VERSION = "4.0";
 
-  public Set<ResolvedDependency> findResolvedDependencies(Project rootProject, boolean allConfigurations) {
-    return new LinkedHashSet<>(rootProject.getAllprojects()).stream()
-        .flatMap(project -> new LinkedHashSet<>(project.getConfigurations()).stream())
+  public Set<ResolvedDependency> findResolvedDependencies(Project project, boolean allConfigurations) {
+    return new LinkedHashSet<>(new LinkedHashSet<>(project.getConfigurations()).stream()
```

Comment:
Why did this change out of curiosity? If I understand correct, previously it was considering the configurations of all projects from `getAllprojects`, and now we're just looking at configurations from the root project?

Reply:
> Why did this change out of curiosity?

Ah nvm I think I understand, looking at the non-test files where it's called
- This is already looping through `rootProject.allprojects` https://github.com/sonatype-nexus-community/scan-gradle-plugin/blob/6728e6f38ec379945b7790d2f4e2cd2e591c13f9/src/main/java/org/sonatype/gradle/plugins/scan/common/DependenciesFinder.java#L96-L113
- You changed this to loop through all `getProject().getAllprojects()` https://github.com/sonatype-nexus-community/scan-gradle-plugin/blob/f2b3cd62add9d7c8fb87e644060e5280f86f4a83/src/main/java/org/sonatype/gradle/plugins/scan/ossindex/OssIndexAuditTask.java#L75-L78

👍